content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -48,33 +48,13 @@ If your current address does not match the address on your ID, you will need to
 
 You can bring a physical copy or show a digital copy of the document. You cannot show a photo or screenshot of the document. 
 
-## Complete the process at the Post Office before your deadline
-
-### Available locations in the Washington D.C. area 
-
-You can visit any of the following Post Office locations. You do not need to visit the location you selected on Login.gov. 
-
-1. **BALTIMORE — Post Office™**
-   900 E FAYETTE ST RM 118
-   BALTIMORE, MD 21233-9715
-2. **BETHESDA — Post Office™**
-   6900 WISCONSIN AVE STE 100
-   CHEVY CHASE, MD 20815-9996
-3. **FRIENDSHIP — Post Office™**
-   4005 WISCONSIN AVE NW
-   WASHINGTON, DC 20016-9997
-4. **WASHINGTON — Post Office™**
-   900 BRENTWOOD RD NE
-   WASHINGTON, DC 20018-9997
-5. **ARLINGTON — Post Office™**
-   3118 WASHINGTON BLVD
-   ARLINGTON, VA 22201-9998
-6. **ASHBURN — Post Office™**
-   44715 PRENTICE DR
-   DULLES, VA 20101-9996
-7. **MERRIFIELD — Post Office™**
-   8409 LEE HWY
-   MERRIFIELD, VA 22116-9998
+## Complete the process at a participating Post Office before your deadline
+
+### In-Person Proofing is available at participating locations across the U.S. and some U.S. territories.*
```

Comment:
I'm a bit surprised it wasn't necessary to escape the asterisk like you'd had to do below to avoid markdown italics, but sure enough, it looks fine in the preview!

Reply:
I also found that surprising